src/ontology/components/breedstatus.owl

```diff
@@ -0,0 +1,199 @@
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">partially recognized</rdfs:label>
+    </owl:Class>
+    <owl:Axiom>
+        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/VBO_0300003"/>
+        <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000115"/>
+        <owl:annotatedTarget rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Breed recognition status referring to a breed that is in the process of becoming fully recognized by an international organization. This status is given to breeds which are preliminarily or provisionally recognized.</owl:annotatedTarget>
+        <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://orcid.org/0000-0002-4142-7153</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://orcid.org/0000-0002-5002-8648</oboInOwl:hasDbXref>
+    </owl:Axiom>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/VBO_0300004 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/VBO_0300004">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/VBO_0300001"/>
+        <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Breed recognition status referring to a breed that is reported by an international organization but not recognized. </obo:IAO_0000115>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">not recognized</rdfs:label>
+    </owl:Class>
+    <owl:Axiom>
+        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/VBO_0300004"/>
+        <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000115"/>
+        <owl:annotatedTarget rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Breed recognition status referring to a breed that is reported by an international organization but not recognized. </owl:annotatedTarget>
+        <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://orcid.org/0000-0002-4142-7153</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://orcid.org/0000-0002-5002-8648</oboInOwl:hasDbXref>
+    </owl:Axiom>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/VBO_0300005 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/VBO_0300005">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/VBO_0300000"/>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">domestication status</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/VBO_0300006 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/VBO_0300006">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/VBO_0300005"/>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">feral</rdfs:label>
```

Comment:
Why not be a good citizen and add definitions to all of these :)

Reply:
(no term without definition, unless self explanatory, like chemical or breed)

Reply:
We are working on these definitions (#44 )